LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 Istio Authors
+   Copyright 2019 Istio Authors
```

Comment:
is this change intended?  Should it not be copyright 2016-2019?

Reply:
That came from the istio/common-files repo. I've fixed the date here, and in the common-files repo so that it will be replicated to all other repos next time the common files are updated.